gravitee-apim-gateway/gravitee-apim-gateway-handlers/gravitee-apim-gateway-handlers-api/src/main/java/io/gravitee/gateway/jupiter/handlers/api/processor/transaction/TransactionPostProcessor.java

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.gateway.jupiter.handlers.api.processor.transaction;
+
+import io.gravitee.gateway.api.http.HttpHeaders;
+import io.gravitee.gateway.jupiter.core.context.MutableExecutionContext;
+import io.gravitee.gateway.jupiter.core.processor.Processor;
+import io.reactivex.rxjava3.core.Completable;
+
+/**
+ * A {@link Processor} used to set the Transaction and Request ID headers of the response.
+ */
+public class TransactionPostProcessor implements Processor {
+
+    private final TransactionPostProcessorConfiguration configuration;
+
+    public TransactionPostProcessor(TransactionPostProcessorConfiguration configuration) {
+        this.configuration = configuration;
+    }
+
+    @Override
+    public String getId() {
+        return "post-processor-transaction";
```

Comment:
I'm not sure we have such a prefix, I only found "processor-something".
should we rename the other ones (as you did with "pre-processor-transaction") to differentiate pre processors from post ones ? (opened question)

Reply:
I did that because this Transaction Processor has 2 parts working "together", one as a pre processor, the other as a post processor. Don't know if there are other processors in that case 🤔